commands/zintercard.md

```diff
@@ -0,0 +1,20 @@
+This command is similar to `ZINTER`, but instead of returning the result set, it returns just the cardinality of the result.
+
+Keys that do not exist are considered to be empty sets.
+With one of the keys being an empty set, the resulting set is also empty (since set intersection with an empty set always results in an empty set).
+
+@return
+
+@integer-reply: the number of elements in the result intersection.
```

Comment:
```suggestion
@integer-reply: the number of elements in the resulting sorted set.
```

Reply:
since it doesn't store a key, i'm not sure it's right to say that the result is "sorted set"
note that this command, same as ZINTERSTORE can also work on `set` types as inputs (probably not documented anywhere)